views/page.html.pug

```diff
@@ -5,30 +5,33 @@ html
     title #{page.name}
     meta(name="viewport" content="width=device-width, initial-scale=1")/
     link(rel="stylesheet" href="//unpkg.com/tailwindcss@1.0.5/dist/tailwind.min.css")/
-    link(href="/"+page.username+".xml" type="application/atom+xml" rel="alternate" title=page.name)/
+    link(href="/"+username+".xml" type="application/atom+xml" rel="alternate" title=page.name)/
```

Comment:
To what `username` refers to here - calmbook logged in user 😉. `page.username` was self explaining

Reply:
Maybe just `username` is not really correct naming and should be more like `path` or `page.path`.

Reply:
Maybe. It feels strange to use 'username' here, but I took it from FB Graph API - [Page reference](https://developers.facebook.com/docs/graph-api/reference/page/) assuming its familiar for anyone who have used Graph API

Reply:
Ok. Fair enough.

Btw, is `username` (alias) something that ALL pages have? Not an expert here, but as far as I remember, you create alias explicitly if you want. Otherwise URL is generated based on the page name suffixed by id.

https://www.facebook.com/gorceultratrail vs https://www.facebook.com/Prognoza-Pogody-Damian-Dąbrowski-216193941873718/

The former has an username (alias) and the latter - most likely - doesn't.

Reply:
Page `username` is optional. If not set page url is build based on page `name` and unique page `id`. 

Maybe we should also add 'search by name' not only by `username`?

Reply:
+ internally only rely on `page.id` not `page.username` nor `page.name`

Reply:
I believe we wrongly use `username` name then as in fact IT IS NOT a username (alias). It's merely url extract, which facebook uses to determine which page to show. Ok, sometimes it is a username (alias), but sometimes it is a page name with id and sometimes maybe it's even something else. But... see the comment above. Let's talk it separately.